lib/python/qmk/cli/console.py

```diff
@@ -0,0 +1,77 @@
+"""hid_listen
+"""
+import hid
```

Comment:
I think this needs `hidapi` in requirements.txt?
Specifically, this one: https://github.com/trezor/cython-hidapi

Reply:
<https://github.com/apmorton/pyhidapi>. Working out if its something that should be hidden under dev, which would place it in the other file.

Reply:
I recall having issues with other libraries, but that might have just been because I didn't have the proper libusb packages installed. May need to check that in the install scripts although dfu-programmer and dfu-util probably pull them in already.

Reply:
Actually, nothing to do with libusb, you need the hidapi package installed:

```
fauxpark@JUNE:~$ pacman -Qs hid
local/mingw-w64-x86_64-hidapi 0.10.1-1
    Library for communicating with USB and Bluetooth HID devices (mingw-w64)
```

Reply:
Sadly doesn't appear to work on macOS 10.15

Reply:
Probably need to `brew install hidapi`.

Reply:
Tried that, and still no go. Even closed and reopened the terminal, just in case

Reply:
Ah, if you've installed the CLI through Homebrew, then you have to hop into the venv and install the Python package there.